docs/platform/manager/using-the-linode-graphical-shell-glish/index.md

```diff
@@ -37,19 +38,17 @@ The process for launching a getty may differ depending on your distribution.
 
 ## Access Glish
 
-1.  From your Linode Dashboard, click on the **Remote Access** Tab. Under **Console Access**, click on the "Launch Graphical Web Console" link:
+1.  From your Linode Dashboard, select the Linode you want to access.
```

Comment:
@Rajakavitha1 It might be better if the user accesses their Linode from the Linodes link in the sidebar, since the dashboard won't show all of a users Linodes.

Reply:
Thank you!! Will make the changes and commit..